src/main/java/org/jitsi/jicofo/JitsiMeetGlobalConfig.java

```diff
@@ -62,7 +62,7 @@
     /**
      * The default value for {@link #PENDING_TIMEOUT_PROP_NAME}.
      */
-    private static final int DEFAULT_PENDING_TIMEOUT = 30;
+    private static final int DEFAULT_PENDING_TIMEOUT = 90;
```

Comment:
Minor observation here for the PENDING_TIMEOUT_PROP_NAME. If this is Jicofo's global config it may be good to have JIBRI or RECORDING as part of the name of the property, for example JIBRI_PENDING_TIMEOUT_PROP_NAME. The same for the default value constant.